pkg/kubelet/config/http.go

```diff
@@ -183,6 +229,63 @@ func tryDecodeList(data []byte) (parsed bool, manifests []v1beta1.ContainerManif
 	return true, manifests, pods, nil
 }
 
+func tryDecodeSinglePod(data []byte) (parsed bool, pod api.Pod, err error) {
+	// TODO(wojtek-t): Should we support pods in different api versions?
```

Comment:
Thanks - I didn't find it and wanted to do it with Convert function (which I didn't know how to do it easily).